perfmetrics/scripts/run_e2e_tests.sh

```diff
@@ -77,20 +80,24 @@ function install_packages() {
   sudo apt install -y python3-crcmod
 }
 
-# Create bucket for integration tests.
 function create_bucket() {
   bucket_prefix=$1
-  # The length of the random string
-  length=5
-  # Generate the random string
-  random_string=$(tr -dc 'a-z0-9' < /dev/urandom | head -c $length)
-  bucket_name=$bucket_prefix$random_string
+  readonly project_id="gcs-fuse-test-ml"
```

Comment:
nit: you can use local -r project_id="gcs-fuse-test-ml". ref https://stackoverflow.com/questions/45409822/bash-local-and-readonly-variable